x-pack/plugins/canvas/public/expression_types/base_form.js

```diff
@@ -3,13 +3,22 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import { injectI18n } from '@kbn/i18n/react';
 
-export class BaseForm {
+class BaseFormUI {
   constructor(props) {
-    if (!props.name) throw new Error('Expression specs require a name property');
+    if (!props.name) {
+      throw new Error(
+        this.props.intl.formatMessage({
+          id: 'xpack.canvas.expressionTypes.namePropertyRequiredErrorMessage',
```

Comment:
xpack.canvas.expressionTypes => xpack.canvas.baseForm for all ids in this file. let us discuss it